osclib/repair_command.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from osc import oscerr
+from osc.core import change_review_state
+from osc.core import get_request
+from osclib.request_finder import RequestFinder
+
+def _is_int(x):
+    return isinstance(x, int) or x.isdigit()
+
+class RepairCommand(object):
+
+    def __init__(self, api):
+        self.api = api
+
+    def repair(self, request):
+        reviews = []
+        reqid = str(request)
+        req = get_request(self.api.apiurl, reqid)
+
+        if not req: 
+            raise oscerr.WrongArgs('Request {} not found'.format(reqid))
+
+        if req.state.name != 'review':
+            print('Request "{}" is not in review state'.format(reqid))
+            return
+
+        for review in req.reviews:
+            if re.match(r'.*:Staging:(.*)$', str(review.by_project)) and review.state == 'new':
+                reviews.append(review.by_project)
+
+        if reviews:
+            if len(reviews) > 1:
+                raise oscerr.WrongArgs('Request {} had multiple review opened by different staging project'.format(reqid))
+        else:
+            print('Request "{}" is not for staging project'.format(reqid))
+
+        staging_project = reviews[0]
```

Comment:
Ops, if `not reviews` is `True` this will cause an exception, maybe we need to `raise oscerr.WrongArgs` too in line 36?